build.gradle

```diff
@@ -142,7 +142,7 @@ subprojects {
     revapi {
       oldGroup = project.group
       oldName = project.name
-      oldVersion = "1.3.0"
+      oldVersion = "1.3.1"
```

Comment:
I don't think we usually update this on patch versions (we haven't done it at least when we released 1.2.1)